src/libraries/System.Net.Quic/ref/System.Net.Quic.cs

```diff
@@ -45,7 +48,26 @@ public sealed class QuicStream : System.IO.Stream
         public override int Read(byte[] buffer, int offset, int count) => throw null;
         public override void Write(byte[] buffer, int offset, int count) => throw null;
         public long StreamId => throw null;
-        public void ShutdownRead() => throw null;
-        public void ShutdownWrite() => throw null;
+        public void AbortRead() => throw null;
+        public ValueTask WriteAsync(ReadOnlyMemory<byte> data, bool endStream, System.Threading.CancellationToken cancellationToken = default) => throw null;
+        public ValueTask ShutdownWriteCompleted(System.Threading.CancellationToken cancellationToken = default) => throw null;
+    }
+    public class QuicClientConnectionOptions
+    {
+        public SslClientAuthenticationOptions ClientAuthenticationOptions { get => throw null; set => throw null; }
+        public IPEndPoint LocalEndPoint { get => throw null; set => throw null; }
+        public IPEndPoint RemoteEndPoint { get => throw null; set => throw null; }
+        public short MaxBidirectionalStreams { get => throw null; set => throw null; }
+        public short MaxUnidirectionalStreams { get => throw null; set => throw null; }
+        public TimeSpan IdleTimeout { get => throw null; set => throw null; }
+    }
+    public class QuicListenerOptions
+    {
+        public SslServerAuthenticationOptions ServerAuthenticationOptions { get => throw null; set => throw null; }
+        public IPEndPoint ListenEndPoint { get => throw null; set => throw null; }
+        public int ListenBacklog { get => throw null; set => throw null; }
+        public short MaxBidirectionalStreams { get => throw null; set => throw null; }
+        public short MaxUnidirectionalStreams { get => throw null; set => throw null; }
```

Comment:
These will likely need to be `long` based on QUIC spec, unless MsQuic does something different.